_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -4,7 +4,45 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
-### Release notes for {{page.version}} 
+#### UI Improvements
+
+Multiple changes have been made in the graphical user interface, thanks to the
+capabilities provided by the new react-based stack. These include a new design
+for the Audits page, improved search bar and pagination, better navigation, and
+many others fixes and improvements.
+
+#### Report templates
+
+A new mechanism has been added that allows generating custom reports.
+See the [Reports](manuals/{{page.version}}/index.html#4.11Reports) section
+for more information.
+
+#### Template rendering engine rewrite
+
+The engine used to render templates has been refactored to allow better
+extendability and testing. Please report any issues found when rendering
+templates after the upgrade.
+
+#### Rails 5.2 upgrade
+
+Foreman has been updated to run on the latest version of Ruby on Rails at the
+time of the release, 5.2.1. Many plugins have already been updated as well, but
+there may be issues with any that have not seen a new release in a while.
+
+#### New system admin role
+
+The purpose of this role is to set up environment for others to use. It can
+create organizations/locations but does not have access to the resources inside
+them. System admin can create new users and assign them to locations or
+organizations and add roles to the users.  See
+[Roles and Permissions](manuals/{{page.version}}/index.html#4.1.2RolesandPermissions)
+for more information.
+
+#### FIPS compliance
+
+Foreman should now run properly in FIPS-enabled environments.
```

Comment:
Should sounds a bit weak, like we're not confident about it. How about `can`?

Reply:
I'll change that in the other PR, thanks.

Reply:
And you are confident? ;-)

Reply:
I didn't test it but I'm confident in the developers who did :)